doc/engineering-reference/src/simulation-models-encyclopedic-reference-003/indoor-swimming-pool.tex

```diff
@@ -141,6 +141,8 @@ \subsection{Evaporation from the pool water surface}\label{evaporation-from-the-
 
 When a cover is present, the cover and the cover evaporation factor reduce the amount of evaporation proportionally. For example, if the pool is half covered and the pool cover reduces convection by 50\%, the convective heat transfer coefficient is reduced by 25\% from the value calculated using the above equation. The value is converted to a latent gain (loss) through multiplication of the evaporation rate by the heat of vaporization of water.
 
+The user should be aware of two key assumptions built into the equations for calculating the evaporation from the pool.  First, when the activity factor is zero, no evaporation will take place.  Thus, activity factor is not the same thing and should not be zero when there are no people in the pool as that will completely eliminate evaporation.  Second, when the cover evaporation factor is zero, the cover will not reduce evaporation at all.  A cover factor of 1.0 means that the cover will completely block evaporation.
```

Comment:
Sorry my proofreading marks weren't clear.  Get me some paper and a red pen so I can use proper editing marks. :-)